source/fundamentals/builders/filters.txt

```diff
@@ -12,34 +12,34 @@ Filters Builders
 
 .. _filters-builders:
 
-Overview 
+Overview
 --------
 
 In this guide, you can learn how to use **builders** to specify
-**filters** for your queries in the MongoDB Java driver.
+**filters** for your queries in the MongoDB Kotlin driver.
 
-Builders are classes provided by the MongoDB Java driver that help you
+Builders are classes provided by the MongoDB Kotlin driver that help you
 construct :ref:`BSON <bson>` objects. To learn more, see our :doc:`guide
-on builders </fundamentals/builders/>`. 
+on builders </fundamentals/builders/>`.
 
 Filters are the operations MongoDB uses to limit your results to what
-you want to see. 
+you want to see.
 
-Some places we use filters are:
+Some places to use filters are:
 
 - ``find()``
 - match stage of aggregation
 - ``deleteOne()``/``deleteMany()``
 - ``updateOne()``/``updateMany()``
 
-Some examples of results from queries with filters are: 
+Some examples of results from queries with filters are:
 
 - Items that cost $0 to $25
 - A hotel with amenities that include an indoor swimming pool and free parking
-- A food critic review that mentions "spicy" 
+- A food critic review that mentions "spicy"
```

Comment:
S: delete this section and better clarify what filters are in the sentence on lines 25-26

Reply:
Let's cut a ticket to improve it here and in the Java page